.github/workflows/update-cdn.yml

```diff
@@ -64,14 +62,20 @@ jobs:
 
         cd cdn
 
-        python -m cdn_generator --bucket-id ${{ secrets.CDN_S3_BUCKET }} --new-release ${{ github.event.client_payload.folder }}/${{ github.event.client_payload.version }}
+        S3_ENDPOINT_URL=${{ secrets.CDN_R2_ENDPOINT }} python -m cdn_generator \
```

Comment:
Funny, it's an R2 but env var is S3.

Reply:
Yup. R2 is S3 compatible; so all scripts think they are talking to an S3, but in reality it is an R2 :)